src/api/Feed.js

```diff
@@ -165,27 +166,38 @@ class Feed extends Base {
     updateAnnotation = (
         file: BoxItem,
         annotationId: string,
-        text: string,
+        text?: string,
+        status?: FeedItemStatus,
         permissions: AnnotationPermission,
         successCallback: (annotation: Annotation) => void,
         errorCallback: ErrorCallback,
     ): void => {
         if (!file.id) {
             throw getBadItemError();
         }
+        if (!text && !status) {
+            throw new Error('Missing text or status!');
```

Comment:
Should this also use `getBadItemError`? Is this message shown to the user? If so, should it be localized in the component rather than passed back here?

Reply:
As mentioned in the previous comment this message is currently not shown to the user.

I was thinking before if this should be the `badItemError` and decided against because this check verifies the updated values of `message` and `status` and those are not yet part of the item. However I'm not strongly against changing it if you think it qualifies as the `badItemError`.